README.md

```
@@ -1,337 +1,67 @@
-    }
-
-    main() {
-    Future.delayed(Duration(seconds: 1))
-     .then(
-     (value) => print(value.a),
-     )
-     .catchError((error)=>print(error));
-    }
+* **Detalhes do Personagem**
 
+    <img width="351" height="765" alt="image" src="https://github.com/user-attachments/assets/efeb886f-93bb-43b0-b5b6-91f679fceebc" />
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)